src/app/details/[media_type]/[id]/page.tsx

```diff
@@ -0,0 +1,42 @@
+import { getDetails } from '@api/getData';
+import Footer from '@components/common/Footer';
+import Play from '@public/icons/play.svg';
+import { Content } from '@api/types';
+
+export default async function Detail({ params }: any) {
+  // 빌드 오류 너무 나서.. 어쩔 수 없이... any...
+  const { media_type, id } = await params;
+
```

Comment:
any 대신에 이렇게 쓰면 오류 안 나지 않으려나 ..?

```suggestion
interface DetailParams {
  media_type: string;
  id: string;
}

interface DetailProps {
  params: DetailParams;
}

export default async function Detail({ params }: DetailProps) {
  const { media_type, id } = params;
```

Reply:
힝 그 때 급하게 이것저것 해도 안 되긴 했었거든.. ㅜㅜ media_type: string; id: string; 이거 정의해도... 근데 그 때 잘못 썼던 걸 수도 있으니까 다음 배포 때 한 번 저렇게 수정해서 해볼게여!